week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/model/Item.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Price } from 'Price'
+
+export interface Item {
```

Comment:
To `Item` razem z `Price` są Twoimi głównymi interfejsami do obsługi modelu danych.
Więc zasługują na osobne pliki - natomiast `Hero` i `CartPanel` to już zwykłe "prop-types".
Mogą mieszkać w tych samych plikach co ich komponenty

Reply:
Zrobione.